test/Chainweb/Test/Utils.hs

```diff
@@ -1043,6 +1054,33 @@ node testLabel rdb rawLogger peerInfoVar conf nid = do
         crs = map snd $ HashMap.toList $ view chainwebChains cw
         poison cr = mempoolAddToBadList (view chainResMempool cr) (V.singleton deadbeef)
 
+withDbDirs :: Word -> ResourceT IO (Map Word (FilePath, FilePath))
```

Comment:
the code to create+destroy the temporary dirs was copied mostly from [System.IO.Temp.withSystemTempDirectory](https://hackage.haskell.org/package/temporary-1.3/docs/src/System.IO.Temp.html#withSystemTempDirectory), but we needed a Map version.